src/sentry/grouping/grouptype.py

```diff
@@ -28,6 +28,8 @@ class ErrorGroupType(GroupType):
     category = GroupCategory.ERROR.value
     default_priority = PriorityLevel.MEDIUM
     released = True
-    detector_validator = ErrorDetectorValidator
-    detector_handler = ErrorDetectorHandler
-    detector_config_schema = {"type": "object", "additionalProperties": False}  # empty schema
+    detector_config = DetectorConfig(
```

Comment:
Could this end up being conflated with the `config` field on detector instances?

Maybe `DetectorSettings` might be less conflated?

Reply:
🤔 i generally agree -- `DetectorSettings` seems alright to me.

it'd be cool if we could name this `Detector` as well, but i think that'd be hyper confusing 🤣